dockernet/config/ica_host.json

```diff
@@ -15,6 +15,7 @@
                     "/cosmos.staking.v1beta1.MsgRedeemTokensForShares",
                     "/cosmos.staking.v1beta1.MsgTokenizeShares",
                     "/cosmos.staking.v1beta1.MsgCancelUnbondingDelegation",
+                    "/cosmos.staking.v1beta1.MsgDisableTokenizeShares",
```

Comment:
Looks like this is enabled on CH - does that sound right?

```
➜  ~ strided q interchain-accounts host params --node https://cosmos-rpc.polkachu.com:443
allow_messages:
- '*'
host_enabled: true
```

Reply:
yup